third_party/blink/renderer/modules/modules_idl_files.gni

```diff
@@ -966,6 +970,7 @@ if (!is_android) {
       get_path_info([
                       "serial/navigator_serial.idl",
                       "serial/worker_navigator_serial.idl",
+                      "input_control/window_input_call.idl",
```

Comment:
Doesn't this need |enable_service_offloading|?

Reply:
Code is modified. Thanks